lib/dock.rb

```diff
@@ -0,0 +1,46 @@
+class Dock
+
+  attr_reader :name,
+              :max_rental_time,
+              :boat_inventory,
+              :total_renters,
+              :total_rental_by_person
+  def initialize(name, max_rental_time)
+    @name = name
+    @max_rental_time = max_rental_time
+    @boat_inventory = {}
+    @total_renters = []
+    @log_hour = 0
+    @total_rental_by_person = {}
+  end
+
+  def add_boats_to_inventory(type)
+    @boat_inventory[type] = type.price_per_hour
+  end
+
+  def log_hour
+    @log_hour += 1
+  end
+
+  def add_renters(renter)
```

Comment:
similar to `inventory` I don't see you using this add_renters method so it is probably overkill